src/setupTests.js

```diff
@@ -1,6 +0,0 @@
-const Enzyme = require('enzyme');
```

Comment:
is this file just empty now? Can delete if so

Reply:
I think I have, it says deleted on the git diff for me! Also didn't realise we had CI on this now, that's handy 😃